src/tests/tests.py

```diff
@@ -19,9 +19,24 @@ def test(bin_dir, dmp_path):
     print('Launching "{0}"..'.format(' '.join(cmd)))
     return subprocess.call(cmd)
 
+def test_python(script_dir, bin_dir, dmp_path, pymodule):
+    py_exe = sys.executable
+    if pymodule == 'kdmp_d':
+        py_exe = py_exe.replace('.exe', '_d.exe')
+    cmd = (
+        py_exe,
+        os.path.join(script_dir, 'tests_bindings.py'),
+        os.path.abspath(bin_dir),
+        dmp_path
+    )
+
+    print('Launching "{0}"..'.format(' '.join(cmd)))
+    return subprocess.call(cmd)
+
 def main():
     parser = argparse.ArgumentParser('Run test')
     parser.add_argument('--bindir', required = True)
+    parser.add_argument('--pymodule', required = True, nargs = '?')
```

Comment:
Can't you just set the argument as `required=False`? I think it'd have the behavior you want

Reply:
I tried and it doesn't work because --pymodule argument is always present (see cmake file) but the value may be empty.